README.md

```diff
@@ -1,8 +1,9 @@
+# Microchip-PIC24-IoT-Wx
```

Comment:
The name of the board is PIC-IoT-Wx where x could be G or A.
https://www.microchip.com/developmenttools/ProductDetails/AC164164